Makefile

```diff
@@ -160,7 +160,7 @@ install-tools: ${TOOLBIN}/protoc-gen-gogo ${TOOLBIN}/etcd
 
 .PHONY: rust-build
 rust-build:
-	FLOW_VERSION=${VERSION} cargo build --release --locked -p bindings
+	FLOW_VERSION=${VERSION} cargo build --release --locked -p bindings -p librocks-exp
```

Comment:
Curious: Why are we only building these two crates as part of the `rust-build`? I assume the other crates get built by being a dependency of the `bindings` crate, but wouldn't the default of building the entire workspace suffice?

Reply:
Yes, I think it would. I honestly don't recall why it is this way... will update